lld/test/ELF/aarch64-reloc-implicit-addend.test

```diff
@@ -1,97 +1,109 @@
-## Test certain REL relocation types generated by legacy armasm.
-# RUN: yaml2obj %s -o %t.o
-# RUN: ld.lld %t.o -o %t
-# RUN: llvm-objdump -s %t | FileCheck %s
+# Test handling of addends taken from the relocated word or instruction
```

Comment:
Missing `REQUIRES aarch64` 

In LLD the convention is to use ## for non executable comments.